samples/README.md

```diff
@@ -0,0 +1,74 @@
+# Data Directory Organization
+
+Each TIDES Data Package example should follow the following directory structure, consistent with the structure of the [Frictionless Data Package specification](https://specs.frictionlessdata.io/data-package/), including:
+
+```
+unique-example-name
+  \TIDES     # Required. Data to be validated against the TIDES specification
+  \datapackage.json # Required. Data package metadata per https://specs.frictionlessdata.io/data-package/
+  \raw      # Optional. Data which the agency uses to create TIDES data
+  \scripts  # Optional. Scripts used to transform raw --> TIDES
+```
+
+## Adding Examples
+
+We encourage the addition of examples, but please follow the following guidelines:
+
+1. *No large files* This isn't the place to store your data, rather to document some minimal examples.  The recommended size is 100-1000 records per file, more if absolutely required to reproduce an issue with the spec.  All individual files should be well under 50 MB.  
+2. *Include Metadata* as specified in [`datapackage.json`](#data-package).
+3. *Include a README.md* in the base folder of your example with an overview so that it can be included in the documentation.
+
+## Data Package
+
+TIDES data packages must include a [`datapackage.json`](https://specs.frictionlessdata.io/data-package/).  Key information to include in [`datapackage.json`](https://specs.frictionlessdata.io/data-package/) includes:
+
+| **Field** | **Description** | **Required** |
+| --------- | --------------- | ------------ |
+| `title` | A human-readable title. | Required |
+| `name` | Identifier string as a URL-friendly slug. | Required |
```

Comment:
Having both `title` and `name` is the most confusing thing to me about the sample data spec. `name` is basically the title, but has the additional property that it must be a url-friendly slug, which is not really defined here. The data package spec only recommends a `name`, `resources` is the only required property. Maybe we can add an explanation of why we require a url-slug and what url-slug means? If we don't intend to use name, because we're not creating a data portal (in the frictionless sense), maybe we can make it optional or remove it?

- name must contain only: lowercase alpha-numeric ASCII characters, underscores (_), en-dashes (-), and periods (.)

Reply:
I added a link to explain slugs and changed the description slightly. I also made it recommended instead of required.